imagersite/imager_images/models.py

```diff
@@ -0,0 +1,67 @@
+"""Models for the Imager Images."""
+
+from django.db import models
+from django.utils import timezone
+from imager_profile.models import ImagerProfile
+
+# Create your models here.
+
+
+class Image(models.Model):
```

Comment:
Specs: "You’ll need to create two models in this new app, the Photo and the Album"

You trouble makers with your alternative naming schemes.  Also, don't forget to use your handy-dandy python 2 compatibility decorators!